js/apps/system/_admin/aardvark/APP/frontend/js/templates/modalGraphTable.ejs

```diff
@@ -67,7 +79,14 @@
 
     <div class="active tab-pane" id="createGraph">
 
-    <div id="smartGraphInfo" class="infoMessage" style="display: none;">
+    <%
+      let smartGraphInfoStyle = "";
+      if (!frontendConfig.isEnterprise) {
+        // In case it we do not have a Enterprise instance, we must hide this by default.
+        smartGraphInfoStyle = "display: none;"
+      }
+    %>
+    <div id="smartGraphInfo" class="infoMessage" style="<%=smartGraphInfoStyle%>">
       <span>SmartGraph Info: Only use non-existent collection names (vertices/edges).</span>
```

Comment:
This should say "EnterpriseGraph Info" or something similar for enterprise graphs.

Reply:
Changed in: 2e5ecba4577c68bb4c1bcec99163689994aac0e7